src/CreateTagsForVSRelease/CreateTagsForVSRelease.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="LibGit2Sharp" Version="0.26.2" />
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="16.153.0" />
```

Comment:
I really like to move all these versions to Versions.props.
However, because all of these projects are using different TFM because of the long history, change starts to blow up.
In future, if we have a chance to sync all projects to Net6.0 then this would be feasible